framework/include/physics/DiffusionPhysicsBase.h

```diff
@@ -23,7 +24,7 @@ class ActionComponent;
  * Base class to host all common parameters and attributes of Physics actions to solve the diffusion
  * equation
  */
-class DiffusionPhysicsBase : public PhysicsBase
+class DiffusionPhysicsBase : virtual public PhysicsBase, virtual public PhysicsComponentInterface
```

Comment:
Virtual inheritance of `PhysicsBase` too now? Why?

Reply:
I see with the inheritance of `PhysicsBase` in `PhysicsComponentInterface`

Reply:
what do you think ? is it worth the additional complexity of virtual inheritance?

Reply:
You're the one developing most in these systems so I think you're the most qualified to make that decision

Reply:
ok I ll keep with virtual inheritance. 
The alternative is templating and I think that s a tad harder to work with though less confusing